README.md

```diff
@@ -83,7 +83,7 @@ uptime-kuma:
 
 ```bash
 
-    TOKEN=$(curl -X -L 'POST' -H 'Content-Type: application/x-www-form-urlencoded' --data 'username=admin&password=admin' http://127.0.0.1:8000/login/access-token/ | jq -r ".access_token")
+    TOKEN=$(curl -L -X 'POST' -H 'Content-Type: application/x-www-form-urlencoded' --data 'grant_type=password&username=admin&password=admin' http://127.0.0.1:8000/login/access-token/ | jq -r ".access_token")
```

Comment:
For me it returns "null".

Reply:
What returns null?

Do you have jq installed?
Did you run the line after as well (replacing the ip address and password with your own)?

```bash
    TOKEN=$(curl -X -L 'POST' -H 'Content-Type: application/x-www-form-urlencoded' --data 'username=admin&password=<YOUR_PASSWORD_GOES_HERE>' http://<DOCKER_IP_GOES_HERE>:<DOCKER_PORT_GOES_HERE>/login/access-token/ | jq -r ".access_token")

    curl -L -H 'Accept: application/json' -H "Authorization: Bearer ${TOKEN}" http://127.0.0.1:8000/monitors/
```

Apologies if it seems I'm saying very obvious things that you already know - I am a n00b myself and I don't know your proficiency with docker/bash/curl.

Reply:
@GavIngram the command works for me.